### ISSUE TITLE
Swipe and hold action.

### ISSUE THREAD

**Issue by Megalomaniak** [keymapperorg/KeyMapper#1288]
We have a swipe action, we have a tap action. They can't be combined to work as a '[swipe to] and then [hold at coordinate] until [trigger release]' action. So maybe a new action type can work instead?

**Developer TODO (don't remove)**
- [ ] create new branch. put issue number at start of name
- [ ] update documentation
- [ ] merge and delete branch (don't squash because want commit history to see why I made changes)

**Comment by typedrat** [comment]
This makes it impossible to use this to play Pokemon: UNITE on my desktop, since there's no way to translate inputs into something the virtual joystick will recognize.

**Comment by jambl3r** [comment]
@typedrat I don't understand what you mean. Could you explain?

**Comment by Megalomaniak** [comment]
@jambl3r if I understand them correct, what they are trying to say is that this would be useful for emulating virtual analog stick inputs. Which is in fact the very issue that inspired me to open this feature request in the first place.

**Comment by typedrat** [comment]
Yeah, I phrased it badly, but that's exactly what I meant. I would like to use KeyMapper to let me play the game with a keyboard and mouse, and without this feature being added, it is not possible to do so.

**Comment by sds100** [comment]
On Android it isn't possible for an accessibility service to tap/swipe the screen in more than one place at once, and if you're finger touches the screen it will cancel the service's taps/swipes. Would this be fine for you?

**Comment by Megalomaniak** [comment]
> Would this be fine for you?

I would say yes. It might not be the ideal, but it's a long way better than having nothing effective or functional at all.

**Comment by sf467** [comment]
+1
